.dockerignore

```diff
@@ -0,0 +1,5 @@
+input/
+output/
+.git*
+assets/
+docs/
```

Comment:
Same question here re newline at end of file.

Reply:
Sure - I can add new lines